.eslintrc.json

```diff
@@ -12,8 +12,10 @@
     {
       "files": ["./src/typeDefs/**/*.ts"],
       "processor": "@graphql-eslint/graphql"
+
     },
     {
+      "extends": ["plugin:@typescript-eslint/disable-type-checked"],
```

Comment:
why is this added?

Reply:
Done a way to resolve the conflicts of parser by disabling type checked in  typeDefs/.ts files as suggested by typescrip-eslint-plugin maintainer  but failed
 Working closely with him to resolve this conflict